src/client/pages/VergunningDetail/ERVV.tsx

```diff
@@ -1,30 +1,37 @@
 import { ERVV as ERVVType } from '../../../server/services';
-import { defaultDateFormat } from '../../../universal/helpers';
+import { defaultDateTimeFormat } from '../../../universal/helpers';
 import InfoDetail, {
   InfoDetailGroup,
 } from '../../components/InfoDetail/InfoDetail';
 import { Location } from './Location';
+import { defaultDateFormat } from '../../../universal/helpers/date';
 
 export function ERVV({ vergunning }: { vergunning: ERVVType }) {
   return (
     <>
       <InfoDetail label="Kenmerk" value={vergunning?.identifier || '-'} />
-      <InfoDetail label="Zaaktype" value={vergunning.caseType || '-'} />
+      <InfoDetail label="Soort vergunning" value={vergunning.caseType || '-'} />
```

Comment:
Dit hebben wij toch, in overleg, een tijdje geleden aangepast naar Zaaktype? Mogelijk is dat niet opgenomen in het design en/of RD. Maar: https://github.com/Amsterdam/mijn-amsterdam-frontend/pull/565 Het was ooit al 'soort vergunning' maar dat hebben we aangepast omdat "Soort vergunning" de lading niet dekt. Bv bij TVM - RVV - Object is het een TVM vergunning, een RVV vergunning óf een Object vergunning maar een vergunning die deze 3 tegelijk verleend bestaat niet.

Reply:
Ja klopt nu je het zegt.
RD was niet aangepast -> Ali viel erover -> ik verward wat we nu hadden afgesproken. Men(en ik volgens mij ook) vond soort vergunnung beter? hmm

Reply:
In eerste instantie (lang geleden) was het 'Soort vergunning' maar dat hebben we toen mondeling afgesproken het aan te passen naar Zaaktype

Reply:
Ff bespreken met Jan Douwe / Ali etc.